source/tests/python/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!@PYTHON_EXECUTABLE@
```

Comment:
`^/timemory/tests/__init__.py`. Also create `^/timemory/tests/__main__.py` so testing can be launched via `python -m timemory.tests <ARGS>`